src/services/raspberrydevconn/settings.json

```
@@ -0,0 +1,12 @@
+{
+    "rest_server": {
+        "host": "0.0.0.0",
+        "port": 8087
```

Already used by DeviceConfig

```suggestion
        "port": 8088
```

---

My fault. Thanks for pointing it out!